modules/nf-core/snpeff/snpeff/main.nf

```diff
@@ -28,13 +28,13 @@ process SNPEFF_SNPEFF {
     if (!task.memory) {
         log.info '[snpEff] Available memory not known - defaulting to 6GB. Specify process memory requirements to change this.'
     } else {
-        avail_mem = task.memory.giga
+        avail_mem = (task.memory.mega*0.8).intValue()
```

Comment:
Can't comment on that line but need to change the `def avail_mem = 6` to megabytes

Reply:
Fixed!